projects/storefrontapp-e2e-cypress/cypress/integration/regression/multisite-isolation/multisite-isolation.core-e2e-spec.ts

```diff
@@ -0,0 +1,109 @@
+import { clearAllStorage } from '../../../support/utils/clear-all-storage';
+import { user } from '../../../sample-data/checkout-flow';
+import { register } from '../../../helpers/auth-forms';
+import * as login from '../../../helpers/login';
+import * as alerts from '../../../helpers/global-message';
+
+import {
+  ELECTRONICS_BASESITE,
+  ELECTRONICS_CURRENCY,
+} from '../../../helpers/checkout-flow';
+import { POWERTOOLS_BASESITE } from '../../../sample-data/b2b-checkout';
+
+export const ELECTRONICS_STANDALONE_BASESITE = 'electronics-standalone';
+export const POWERTOOLS_STANDALONE_BASESITE = 'powertools-standalone';
+
+describe('Multisite Isolation', () => {
+  before(() => {
+    cy.window().then((win) => {
+      win.localStorage.clear();
+    });
+  });
+
+  beforeEach(() => {
+    clearAllStorage();
+  });
+
+  describe('Standalone authorization', () => {
+    beforeEach(() => {
+      Cypress.env('BASE_SITE', ELECTRONICS_STANDALONE_BASESITE);
+
+      cy.cxConfig({
+        context: {
+          baseSite: [ELECTRONICS_STANDALONE_BASESITE],
+          currency: [ELECTRONICS_CURRENCY],
+        },
+      });
+    });
+
+    it.skip('should register customer on standalone environment', () => {
+      cy.visit('/login/register');
+      register(user);
```

Comment:
Do we _really_ want to "register" an user credentials that is shared among possibly more test suites (that may also want to possibly register this user)?

Although maybe everything works fine, the consequences of using a shared user credentials are not clear at the first sight. For clarity, what about creating fresh user credentials that are shared only in the scope of this test suite?

Reply:
For check authentication we need first to create a user on specific standalone basesite. I am not sure if we can use pre-defined for testing such cases (users for electronics-spa and powertools-spa won't work). I need to investigate.

As far as I checked `uid` is shared through all scenarios I prepared in the test suite.